apps/web/vibes/soul/sections/products-list-section/sorting.tsx

```diff
@@ -58,6 +58,7 @@ function SortingInner({
 
```

Comment:
This wasn't changed in the PR, but do you know why we don't use a default value for `label` in the prop definition instead of giving the default here? So

Reply:
Because the value might be a promise that resolves to `null` so defaulting the prop when destructuring it would only handle the case when the prop isn't passed in, not the `null` case